src/arc_utilities/rviz_marker_repeater.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python
```

Comment:
I am not going to review this file in detail right now.

I prefer to make custom rviz marker types, so I probably wont use this.
An example and more explanation would be nice.

Feel free to resolve without change